src/app/effects/work-item.effects.ts

```diff
@@ -242,13 +242,15 @@ export class WorkItemEffects {
 
         // We don't update work item type
         // So we remove it from the payload
-        const payload = cleanObject({
+        const payload = {
```

Comment:
Remove the comments above.

Reply:
Besides cleaning the properties, mentioned in second arg array, `cleanObject` also clean up the properties with value null. So we'd still need that function. Just remove the second arg, i.e. `['baseType']` part from it.

Reply:
Sure